l10n_ec_account_edi/data/email_template_unauthorized_notify.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_unauthorized_notify" model="mail.template">
+        <field name="name">Unauthorized electronic documents</field>
+        <field name="model_id" ref="model_account_edi_document" />
+        <!--        <field name="model_id" ref="model_res_company" />-->
+        <field name="subject">Unauthorized electronic documents</field>
+        <field name="email_to">somebody@odoo.ec</field>
+        <field name="body_html" type="html">
+            <div>
+                Dear,
+                <br />
+                This email is to notify you that the documents electronics
+                are unauthorized.
+                <br />
+            </div>
+
+            <t
+                t-set="records"
+                t-value="object.env['account.edi.document'].search([('state', '=', 'to_send'), ('move_id.company_id.id', '=', object.id)])"
```

Comment:
It's better to implement this code in a Python function for easier inheritance in other modules if necessary.